devices/stm32h7x3.yaml

```diff
@@ -321,6 +321,97 @@ Ethernet_DMA:
       INTM:
         bitWidth: 2
 
+RCC:
+  _modify:
+    CIFR:
+      access: read-only
+  CR:
+    _modify:
+      RC48ON:
+        name: HSI48ON
+      RC48RDY:
+        name: HSI48RDY
+  CRRCR:
+    _modify:
+      RC48CAL:
+        name: HSI48CAL
+  CFGR:
+    _modify:
+      MCO1SEL:
+        name: MCO1
+      MCO2SEL:
+        name: MCO2
+  CIER:
+    _modify:
+      RC48RDYIE:
+        name: HSI48RDYIE
+  CIFR:
+    _modify:
+      RC48RDYF:
+        name: HSI48RDYF
+  CICR:
+    _modify:
+      RC48RDYC:
+        name: HSI48RDYC
+  BDCR:
+    _modify:
+      VSWRST:
+        name: BDRST
+      RTCSRC:
+        name: RTCSEL
+  APB1LRSTR:
+    _modify:
+      HDMICECRST:
+        name: CECRST
+  D3AMR:
+    _modify:
+      BKPSRAMAMEN:
+        name: BKPRAMAMEN
+  APB1LENR,C1_APB1LENR:
+    _modify:
+      HDMICECEN:
+        name: CECEN
+  AHB1ENR,C1_AHB1ENR:
+    _modify:
+      USB1OTGEN:
+        name: USB1OTGHSEN
+      USB2OTGEN:
+        name: USB2OTGHSEN
+      USB1ULPIEN:
+        name: USB1OTGHSULPIEN
+      USB2ULPIEN:
+        name: USB2OTGHSULPIEN
+        bitOffset: 18
+  AHB2ENR,C1_AHB2ENR:
+    _modify:
+      CAMITFEN:
+        name: DCMIEN
+        description: "DCMI peripheral clock"
+  APB1LLPENR,C1_APB1LLPENR:
+    _modify:
+      HDMICECLPEN:
+        name: CECLPEN
```

Comment:
This was done in the upstream SVD.

Reply:
I think this one was actually missed in both `APB1LLPENR` _and_ `C1_APB1LLPENR`